### ISSUE TITLE
v1.7 alpha release No error being displayed

### ISSUE THREAD

**Issue by TnCoders** [UniSharp/laravel-filemanager#188]
Hi,

using v1.7 alpha release when I upload picture bigger then  the lfm config file :  'max_image_size' => 500,

Server response is OK like there is no problem and there is no error displayed for the user.

Of corse the picture was not uploaded because it's bigger than 500kb

**Comment by g0110280** [comment]
Is there any error in console? If there is a 413 error, check your nginx setting.

**Comment by TnCoders** [comment]
Im using Apache not Nginx .

The response was "OK"  but the file was not uploaded.

The file was 3mb size and File limit size 1mb

**Comment by youchenlee** [comment]
This is because when hitting the [error](https://github.com/UniSharp/laravel-filemanager/blob/master/src/traits/LfmHelpers.php#L304-L307), it return `trans()` where it's a message with HTTP 200 OK.

I think this is a bug, we must mapping these errors to proper HTTP response codes.

**Comment by g0110280** [comment]
This is not a bug, I think it is about web server configuration.
When uploading larger files like photos taken by smartphones, Apache responses 200 OK and Nginx responses 413 request too large, before reaching PHP.

**Comment by youchenlee** [comment]
I meant in the php codes, we should not return 200 ok when error happens. This is nothing to do with nginx nor apache.

**Comment by youchenlee** [comment]
This only happens on mobile browser? @g0110280

**Comment by g0110280** [comment]
Considering multiple files, uploading process should not be interrupted by validation. Files that passed the validation should be uploaded, and inform user that some files are not uploaded. Thus the filemanager will 200 OK with error message.

**Comment by g0110280** [comment]
The situation I mention above is tested on Windows machine with Apache. Tested on PC browser and didn't test on mobile browser.

**Comment by youchenlee** [comment]
@g0110280 got your point, thanks :)

@TnCoders would you provide some relative apache and php configs.

Ex, 
post_max_size and upload_max_filesize

Also, what is the http status code while you get the "OK" response?

**Comment by stormhillmedia** [comment]
I am not sure if this is also server related but I see a similar issue but using Laravel Homestead on Vagrant on a Win10 machine using Firefox or Chrome. In my case I can see that an error is generated but it does not output onto the screen.

E.G. 
1. I tried uploading the same file twice. The second time it failed to upload (as expected) but no error displayed

2.  in UploadController line 96ff  I changed
        if (File::exists(parent::getCurrentPath($new_filename))) {
            return $this->error('file-exist');
        }

TO 

        if (File::exists(parent::getCurrentPath($new_filename))) {
            echo $this->error('file-exist');
        }
and now the error outputs on the screen in a popup, followed by the word OK.
